src/grt/test/pd1.tcl

```diff
@@ -0,0 +1,27 @@
+# layer range for clock nets. def file from the openroad-flow (modified gcd_sky130hs)
+source "helpers.tcl"
+read_liberty "sky130hs/sky130hs_tt.lib"
+read_lef "sky130hs/sky130hs.tlef"
+read_lef "sky130hs/sky130hs_std_cell.lef"
+
+read_def "clock_route.def"
+
+current_design gcd
+create_clock -name core_clock -period 2.0000 -waveform {0.0000 1.0000} [get_ports {clk}]
+set_propagated_clock [get_clocks {core_clock}]
+
+set guide_file [make_result_file pd1.guide]
+
+set_pdrev_alpha 0.3
```

Comment:
not sure what this tests when .3 is the default

Reply:
If I set alpha as 0.3 by default, grt will use pd, but I want flute as the default tree constructor. Should I set alpha as 0.3 and call `set_routing_alpha 0` in all grt scripts or set alpha as 0.0 and call `set_routing_alpha 0.3` in resizer scripts?

Reply:
Why do you want to make flute the default?
The push is to switch to PD because of better results.